src/Sarif.Multitool/ExportCommand.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+using Microsoft.CodeAnalysis.Sarif.Driver;
+using Microsoft.CodeAnalysis.Sarif.Multitool.Rules;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    public class ExportCommand : CommandBase
```

Comment:
>ExportCommand [](start = 17, length = 13)

Let's call this `ExportRuleDocumentation`. "Export" is such a generic term, you might be exporting anything. You could imagine an `ExportRules` command that creates a SARIF file containing the rule metadata.